src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -92,13 +94,17 @@ function WorkspaceInitialPage({policyDraft, policy: policyProp, route}: Workspac
     const [cardsList] = useOnyx(`${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}${workspaceAccountID}_${CONST.EXPENSIFY_CARD.BANK}`);
     const [connectionSyncProgress] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY_CONNECTION_SYNC_PROGRESS}${policy?.id}`);
     const [policyCategories] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY_CATEGORIES}${route.params?.policyID ?? '-1'}`);
+    const [personalDetails] = useOnyx(ONYXKEYS.PERSONAL_DETAILS_LIST);
+    const currentUserPersonalDetails = useCurrentUserPersonalDetails();
```

Comment:
There is another `useCurrentUserPersonalDetails` hook, let's remove it

Reply:
I updated.